Integrations/rasterize/rasterize.yml

```diff
@@ -3,6 +3,11 @@ commonfields:
   id: Rasterize
   version: -1
 configuration:
+- display: 'Throw Errors '
```

Comment:
change to 'Return Errors' and add in the detailed description that explains that it will return warning entries instead of error entry when unmarked.